CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
 Releases
 ========
 
-v1.1.0-dev (unreleased)
+v1.0.1-dev (unreleased)
 -----------------------
 
--   No changes yet.
+-   Bugfix: Thrift: Code generated for empty services would fail to compile.
```

Comment:
We haven't been using any convention for bugfixes. I was looking for something
better than "Fixed a bug where X happened". Other options:

    -   Bug: ...

    -   Fixed code generation for empty services.
    
I'm open to suggestions.